go-controller/pkg/ovn/ovn.go

```diff
@@ -216,7 +228,11 @@ func (oc *DefaultNetworkController) ensureRemoteZonePod(oldPod, pod *kapi.Pod, a
 			return fmt.Errorf("addPodExternalGW failed for remote pod %s/%s: %v", pod.Namespace, pod.Name, err)
 		}
 	}
-	return nil
+	switchNames, err := oc.getSwitchNames(pod)
+	if err != nil {
+		return fmt.Errorf("failed retrieving switch names when ensuring remote zone for pod %s/%s: %v", pod.Namespace, pod.Name, err)
+	}
+	return kubevirt.EnsureRoutingForVM(oc.controllerName, oc.watchFactory, oc.nbClient, switchNames, false /* remote zone */, pod, oc.GetNetworkName())
```

Comment:
Why are we not checking here if `IsPodLiveMigratable` symmetrically to what we do in `ensureLocalZonePod`?

Reply:
I will check it.